packages/react/src/utils/index.js

```diff
@@ -43,8 +52,8 @@ export const getApiUrl = ({
   data,
   force,
   headers,
+  iframe,
   media,
-  prerender = 'auto',
```

Comment:
Just out of interest, how come `prerender` has been removed?

Reply:
I removed it because 'prerender=auto' is the default API value

Reply:
Ah right, but then you should just remove the `= 'auto'` part right? We still want to pass `prerender` to `getApiUrl`.

Reply:
oh, totally true!